README.md

```diff
@@ -2,284 +2,64 @@
 
-Install dependencies:
-```
-poetry install
-```
+![get projects task mappings execute panel](./docs/images/project_execute.png)
 
-#### Prerequisites
-You will need to set your Evergreen credentials in environment variables because the application
-and CLIs authenticate to Evergreen using $EVG_API_USER and $EVG_API_KEY. You can set these to the
-values in your local ~/.evergreen.yml file. You will also need to set the location of a mongodb
-instance in an environment variable called ```SELECTED_TESTS_MONGO_URI```:
-```
-export EVG_API_USER="<your evg api user>"
-export EVG_API_KEY="<your evg api key>"
-export SELECTED_TESTS_MONGO_URI="localhost:27017"
-uvicorn --host 0.0.0.0 --port 8080 --workers 1 selectedtests.app.asgi:app <--reload>
-```
-__Note__: reload is only used in development mode.
+Click the blue **Execute** button below the parameters panel.
 
-## Testing/Formatting/Linting
-```
-poetry run black src tests
-poetry run pydocstyle src
-poetry run pytest
-```
+The response body field on the panel will display the test or task mappings discovered for the project and files entered (you
+can see some of the results at the bottom of the previous image).
 
-To get code coverage information, you can run pytest directly.
-```
-$ poetry run pytest --cov=src --cov-report=html
-```
+This would be the equivalent of changing only the **src/mongo/db/index_builds_coordinator.cpp** file
+in a mongo branch and issuing the `evergreen patch` command(above). 
 
-## Merging code to master
+You can also view the mappings from the command line. See [Curl Mappings](docs/ManuallyRunning.md#curl-mappings) for more details. 
+If you are trying to access the MongoDB service then you will also need to [Authenticate](docs/MongoService.md#authentication).
 
-Merges to the selected-tests repo should be done via the Evergreen [Commit Queue](https://github.com/evergreen-ci/evergreen/wiki/Commit-Queue).
+If any new endpoints are added to the service or if the service is updated in such a way that any of
+the existing endpoints' contracts change, the swagger documentation must be updated to reflect the
+new state of the service before that change can be merged to master.
 
-When your PR is ready to merge, add a comment with the following:
-```
-evergreen merge
-```
+Documentation for how the swagger documentation is done can be found
+[here](https://flask-restplus.readthedocs.io/en/stable/swagger.html).
 
-## Deploy
+# Further Reading 
 
-Deployment is done via helm to [Kanopy](https://github.com/10gen/kanopy-docs#index) (MongoDB
-internal service). The project will automatically be deployed on merge to master. The deployed
-application can be accessed at
-https://selected-tests.server-tig.prod.corp.mongodb.com/health (MongoDB internal
-app, see Authentication section for access).
+More detailed about the Selected Test service is contained in the following: 
```

Comment:
I like the way that you split these files up, I know it's a lot to organize.

Maybe we could add some information about the intended audience for each sub-readme? i.e. `ManuallyRunning.md: For an engineer outside MongoDB who wants to run their own selected-tests service` etc.